elasticdl/python/master/task_dispatcher.py

```diff
@@ -134,6 +142,45 @@ def create_tasks(self, task_type, model_version=-1):
                 self._todo.extend(tasks)
         return tasks
 
+    def _create_save_model_task(self):
+        # For the SavedModel Task, we need a shard of data to build the model
```

Comment:
Add a docstring and place the not in the docstring.
"we need a shard of data to build the model" -> "we need create a dataset with a shard of data to build the model"

Reply:
Good suggestion, updated.